beniget/beniget.py

```diff
@@ -241,7 +245,9 @@ def dump_chains(self, node):
 
     def unbound_identifier(self, name, node):
         if hasattr(node, "lineno"):
-            location = " at {}:{}".format(node.lineno, node.col_offset)
+            location = " at {}{}:{}".format(self.filename,
```

Comment:
Some extra thoughts: if that becomes part of the library, we should provide a way for end-users to override this, e.g. through subclassing. This means we should probably provide a `def print_unbound_identifier(self, name, node)` method. And pretty print the case when `self.filename is None` in a decent way too.

Reply:
Uh... isn't that just `unbound_identifier` here? All it does is print the message, end-users can already override this.

Reply:
As for handling `None`, you mean move the logic from `__init__` to this method?

Reply:
> Uh... isn't that just `unbound_identifier` here? All it does is print the message, end-users can already override this.

ahah right, forget about this stupid remarks.

Reply:
> As for handling `None`, you mean move the logic from `__init__` to this method?

More like

```
filename = "<unknown>" if self.filename is None else self.filename
location = " at {}{}:{}".format(filename, ...)
```

Reply:
Done!